.github/workflows/publishDocumentation.yml

```diff
@@ -0,0 +1,30 @@
+name: PublishDocumentation
```

Comment:
nitpick: Spaces are allowed, and other Actions utilize spaces, so might as well do the same to be consistent.

e.g.
![Screen Shot 2020-12-14 at 7 28 18 PM](https://user-images.githubusercontent.com/98017/102167114-95fbc180-3e42-11eb-89b2-48de95550f76.png)

```suggestion
name: Deploy Documentation
```